server/src/controllers/questionsController.ts

```diff
@@ -47,3 +47,45 @@ export const addQuestion = async (req: Request, res: Response) => {
     res.status(500).json(error);
```

Comment:
```suggestion
    res.status(500).json({ error: "Server Error" })
```

please while we are here anyway 😁